### PR TITLE
Integrating discussion landing

### LINKED ISSUE #11: Integrating Discussion and Landing Page
Currently the Discussion page is a separate page, integrating it with discussion page hence different discussion page for different topics and chats.

### PR DESCRIPTION
**Pull request by DevanshCodes**
Features Added:

- Separate Pages for each topic Discussion.

- Single Page website, with component Discussion

- Everything is being read from JSON

- Added the Reply feature

### CODE REVIEW

**Comment by DevanshCodes** [comment]
@ayushnagar123

File: src/components/sidebar/sidebar.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@
   top: 0;
   bottom: 0;
   left: 0;
-  min-height: 93.5vh;
+  min-height: 92.5vh;
```
Comment (ayushnagar123) [inline src/components/sidebar/sidebar.scss:11]: 1. need to fix the hight issue the height of the navbar looks in appropriate.
image

2. Navbar content overlaps below 1050px. Responsiveness needs to be taken into handle.
image

**Review by ayushnagar123 — CHANGES_REQUESTED** [review]
image
Errors while render of each page must be handled.

File: src/components/sidebar/sidebar.scss (hunk 1 above)
Comment (DevanshCodes) [reply src/components/sidebar/sidebar.scss:11]: Cool will work on that!

**Comment by DevanshCodes** [comment]
Fixed all issues, just that error of unique key is left, I'll fix that in my other PR coz this will cause huge merging error in my new PR which is already there. 
Ready for merging.

**Pull request merged**